### PR TITLE
FUSETOOLS-2876 - Adapt UI tests to changes introduced in 10.3.0.AM3

### PR DESCRIPTION
**Pull request by tsedmik**
https://issues.jboss.org/browse/FUSETOOLS-2876

Signed-off-by: Tomáš Sedmík <tsedmik@redhat.com>

# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [x] Did you use the Jira Issue number in the commit comments?
- [x] Did you set meaningful commit comments on each commit?
- [x] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [ ] Did the CI job report a successful build?
- [ ] Is the non-happy path working, too?

## Maintainability

- [ ] Are all Sonar reported issues fixed or can they be ignored?
- [ ] Is there no duplicated code?
- [ ] Are method-/class-/variable-names meaningful?

## Tests

- [ ] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?
- [ ] Do we need a new UI test?

## Legal

- [ ] Have you used the correct file header copyright comment?
- [ ] Have you used the correct year in the headers of new files?

### CODE REVIEW

**Comment by tsedmik** [comment]
testPR

File: uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/utils/ProjectFactory.java [hunk 1]
```diff
@@ -58,44 +67,47 @@ public ProjectFactory version(String version) {
...
-			wiz.selectTemplate(template);
+		NewFuseIntegrationProjectWizardAdvancedPage lastPage = new NewFuseIntegrationProjectWizardAdvancedPage(wiz);
+		if (template != null) {
+			lastPage.selectTemplate(template);
+		} else {
+			lastPage.selectTemplate(SPRINGBOOT);
 		}
-		wiz.setProjectType(type);
-		new FinishButton().click();
+		new FinishButton(wiz).click();
```
Comment (apodhrad) [inline uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/utils/ProjectFactory.java]: You can use the finish method, e.g.
```
wiz.finish( TimePeriod.getCustom(900));
```

**Comment by tsedmik** [comment]
All comments will be processed in this JIRA issue https://issues.jboss.org/browse/FUSETOOLS-2893

**Comment by tsedmik** [comment]
testPR

**Comment by apupier** [comment]
it was ready for merge when master would have been green as usual.
Anyway, keep it like that this time.

**Pull request merged**